### PR TITLE
Add icon for PTRN token

### PR DESCRIPTION
**Pull request by KoliosterNikolayIliev**
### 📝 Description

_Add logo for PTRN tokens to ledger. Support for tokens was added to ledger live desktop already_
Contract address (polygon):
PTRN - 0x324165Db0A8D41f2eEBb38d68E75Edd5F8f48963
### ❓ Context

- **Impacted projects**: `live-desktop, @ledgerhq/live-common` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes**

### CODE REVIEW

**Comment by KoliosterNikolayIliev** [comment]
Hi @aelmilligy-ledger,
I think now should be Ok.
Thanks!

**Comment by KoliosterNikolayIliev** [comment]
Hi, @gre,

Please, advise how to proceed. Should I close the PR, delete the branch and start the process from the beginning?

**Comment by KoliosterNikolayIliev** [comment]
I hope now the commit history is OK.

File: libs/ledger-live-common/src/data/icons/svg/PTRN.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="Layer_2" data-name="Layer 2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.33 23.28">
+  <defs>
+    <style>
+      .cls-1 {
+        fill: #273658;
+      }
+    </style>
+  </defs>
+  <g id="Layer_1-2" data-name="Layer 1">
```
Comment (gre) [inline libs/ledger-live-common/src/data/icons/svg/PTRN.svg:10]: we must not have id or class in the SVG. these will not work for React Native integration.
see other icons for example.
There might be an automatic tool out there to sanitize the SVG but we must not have defs, style, classes or global ids.

Thanks

Comment (KoliosterNikolayIliev) [reply libs/ledger-live-common/src/data/icons/svg/PTRN.svg:10]: Thanks for the info. I will fix it and make a new commit.

**Comment by KoliosterNikolayIliev** [comment]
Hi, I do not understand, why all test checks are failing. There is no code change in this PR?
Thanks.

**Comment by haammar-ledger** [comment]
> Hi, I do not understand, why all test checks are failing. There is no code change in this PR. Thanks.

Hi @KoliosterNikolayIliev, this test checks that the provided SVG respects some rules: https://github.com/LedgerHQ/ledger-live/actions/runs/5762407712/job/15622111364#step:8:5636
If you want to have a look.

**Review by gre — APPROVED** [review]

**Review by chabroA — APPROVED** [review]

**Review by alexandremgo — APPROVED** [review]

**Pull request merged**